### PR TITLE
samples: coremark: make ppr run from ram

### PR DESCRIPTION
**Pull request by kapi-no**
Changed the CoreMark sample to use the PPR core configuration that uses RAM for code execution (the non-XIP variant).

Ref: NCSDK-22397

### CODE REVIEW

**Comment by kapi-no** [comment]
This change is needed as the logging subsys gets stuck on the PPR core in the XIP configuration and this sample becomes unusable for the PPR core.

**Review by divipillai — APPROVED** [review]

File: samples/benchmarks/coremark/Kconfig.sysbuild [hunk 1]
```diff
@@ -20,6 +20,6 @@ config APP_CPUPPR_RUN
 	  benchmark only on two cores at a time; application core and either radio core or PPR
 	  core.
 
-	  PPR code is run from MRAM with the execution in place (XIP) method. You must use the
-	  "nordic-ppr-xip" snippet for the application core to be able to boot the PPR core.
-	  You need to provide the "coremark_SNIPPET=nordic-ppr-xip" build argument.
+	  PPR code is run from RAM. You must use the "nordic-ppr" snippet for the application
```
Comment (zycz) [inline samples/benchmarks/coremark/Kconfig.sysbuild:23]: Maybe lets inform exactly that this is tightly couple memory for ppr - ram RAM30 (TCM for PPR).

File: samples/benchmarks/coremark/README.rst [hunk 2]
```diff
@@ -157,14 +145,14 @@ SB_CONFIG_APP_CPUPPR_RUN - Enable execution for the PPR core
    Enable the benchmark execution also for the PPR core for targets with the nRF54H20 SoCs.
 
 .. note::
-   PPR code is run from MRAM with the execution in place (XIP) method.
-   You must use the ``nordic-ppr-xip`` snippet for the application core to be able to boot the PPR core.
-   Use the build argument ``coremark_SNIPPET=nordic-ppr-xip``.
+   PPR code is run from RAM.
```
Comment (zycz) [inline samples/benchmarks/coremark/README.rst:148]: same as above - RAM30 (TCM for PPR).

**Review by zycz — APPROVED** [review]

File: samples/benchmarks/coremark/README.rst (hunk 2 above)
Comment (kapi-no) [reply samples/benchmarks/coremark/README.rst:148]: We did not provide such information for MRAM in the XIP mode (the exact MRAM block number that is used for the specific core). Getting into the platform-specific details here would make this documentation harder to maintain

File: samples/benchmarks/coremark/Kconfig.sysbuild (hunk 1 above)
Comment (kapi-no) [reply samples/benchmarks/coremark/Kconfig.sysbuild:23]: We did not provide such information for MRAM in the XIP mode (the exact MRAM block number that is used for the specific core). Getting into the platform-specific details here would make this documentation harder to maintain

File: samples/benchmarks/coremark/README.rst (hunk 2 above)
Comment (kapi-no) [reply samples/benchmarks/coremark/README.rst:148]: I can provide information regarding the TCM if you believe that using it for PPR will be standard approach for all platforms

**Review by alstrzebonski — APPROVED** [review]

**Pull request merged**